.gitignore

```diff
@@ -9,6 +9,7 @@
 /log/*.log
 /tmp
 /.sass-cache
+/.byebug_history
```

Comment:
Nah it depends on the order, you need to add it to your `gitignore`, then the `gitignore` needs to be in staging, THEN the file gets ignored. I think.